tidb-cloud/data-service-get-started.md

```diff
@@ -58,22 +58,19 @@ If you want to create a new endpoint, locate the newly created Data App and clic
 
 On the right pane, click the **Properties** tab and set properties for the endpoint, such as:
```

Comment:
The following changes simplify the get started doc as these limitations will be prompted if users input an invalid value.